### PR TITLE
Fixes issue where DZ was still creating thumbnail for tif files.

### PR DESCRIPTION
**Pull request by bseeger**
https://www.pivotaltracker.com/story/show/183995116

We had missed spot where we needed to check to see if the mimetype was a tiff file.   

To test, simply upload a tif file.  See that it actually completes and displays the default icon.

### CODE REVIEW

File: src/main/resources/templates/fragments/fileUploader.html [hunk 1]
```diff
@@ -273,7 +275,7 @@
           if (!file.type.includes("image")) {
             setDefaultThumbnail(file);
           }
-          if (file.type.includes("image/heic")) {
+          if (file.type.includes("image/heic") || file.type.includes("image/tif")) {
```
Comment (bseeger) [inline src/main/resources/templates/fragments/fileUploader.html:278]: The is the only real change. The rest are formatting changes intellij did.

**Comment by bseeger** [comment]
Hmmm.... still seeing multiple files not finish. Sigh.

**Comment by bseeger** [comment]
Apparently there is still a stop missing because it fails if a tiff is part of a multi file upload. Sigh.

**Comment by bencalegari** [comment]
Alex and I merged changes to move dropzone so we're gonna close this here and reopen it in form-flow

**Pull request closed without merging**